examples/psyad/eg2/README.md

```diff
@@ -51,7 +51,7 @@ make
    test-harness code and the TL and adjoint kernels:
 ```sh
 cd <lfric-root>/miniapps/skeleton
```

Comment:
As you commented, this is out of date now. As a minimal fix, please could you change L41 to talk about 'LFRic core' and then replace 'miniapps' here with 'applications' (and at L39, L69 and L78).